pkg/runtime/wfengine/backend.go

```diff
@@ -106,10 +108,21 @@ func (be *actorBackend) CreateOrchestrationInstance(ctx context.Context, e *back
 		WithContentType(invokev1.OctetStreamContentType)
 	defer req.Close()
 
+	start := time.Now()
 	resp, err := be.actors.Call(ctx, req)
+	elapsed := diag.ElapsedSince(start)
 	if err != nil {
+		// Failed request to create workflow, record metrics.
+		diag.DefaultWorkflowMonitoring.WorkflowOperationsCount(ctx, diag.CreateWorkflow, diag.StatusFailed)
+
+		// record latency metrics.
+		diag.DefaultWorkflowMonitoring.WorkflowOperationsLatency(ctx, diag.CreateWorkflow, diag.StatusFailed, elapsed)
```

Comment:
Similarly you can combine both into a common event function: https://github.com/dapr/dapr/blob/5050df5bf10835e9890b6b6519f5fa22e4736b28/pkg/diagnostics/component_monitoring.go#L210